src/mongo-read-stream.ts

```diff
@@ -79,13 +80,17 @@ export class MongoReadStream<
 		private readonly projection: object = {},
 		private readonly customType?: ClassType<Partial<U | L>>,
 		private readonly hint?: string | object,
+		private readonly sort: object = { _id: 1 },
```

Comment:
After trying a few things and giving it thoughts, is would not be possible to easily adapt the query with the sort param:

Imagine we have the following objects in our collection:
```
{ key: '1', value: '1' }
{ key: '1', value: '2' }
{ key: '2': value: '1' }
{ key: '2': value: '2' }
{ key: '3': value: '1' }
{ key: '3': value: '2' }
```

and the following sort: { key: 1, value: 1 }
and the following pagination size: 3

The first iteration would get the first 3 items without issues.
However, in the next iteration we would have the following query:
```
{
   $and: [
      { key: { $gt: '2' } },
      { value: { $gt: '1' } },
   ]
}
```
 
Which will end up only fetching the last object { key: '3': value: '2' } because it's the only meeting the query conditions.

Since I don't want to spend too much time on this, I suggest we remove the sort param and only sort by _id as it is not needed for now, what do you think ?